bundles/org.openhab.binding.homewizard/src/main/java/org/openhab/binding/homewizard/internal/devices/HomeWizardDeviceHandler.java

```diff
@@ -182,12 +194,14 @@ private boolean processDeviceInformation() {
             deviceInformation = getDeviceInformationData();
         } catch (Exception ex) {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
-                    String.format("Unable to query device: %s", ex.getMessage()));
+                    "@text/offline.comm-error-device-offline");
+            logger.error("Unable to get device information", ex);
```

Comment:
As the thing offline is already logged including a message, the stacktrace should only be logged in debug
```suggestion
            logger.debug("Unable to get device information", ex);
```